process/Code review process.asciidoc

```diff
@@ -0,0 +1,288 @@
+= Code review process
+
+:toc:
+:toclevels: 4
+
+:numbered!:
+[abstract]
+== Summary
+Description of the code review process using by the log2timeline projects
+
+[preface]
+== Document information
+[cols="1,5"]
+|===
+| Author(s): | Joachim Metz <joachim.metz@gmail.com>
+| Abstract: | Description of the code review process using by the log2timeline projects
+| Classification: | Public
+| Keywords: | Design proposal
+| Status: | Draft
+|===
+
+[preface]
+== License
+....
+Copyright (C) 2016, log2timeline projects maintainers <log2timeline-maintainers@googlegroups.com>
+Permission is granted to copy, distribute and/or modify this document under the
+terms of the GNU Free Documentation License, Version 1.3 or any later version
+published by the Free Software Foundation; with no Invariant Sections, no
+Front-Cover Texts, and no Back-Cover Texts. A copy of the license is included
+in the "LICENSE" file.
+....
+
+:numbered:
+== Introduction
```

Comment:
I meant moving all the background, documentation of the old process and motivation to the end, and focusing on what we currently do right now.